docs/xls-38d-cross-chain-bridge/ledger-object-types/xchainownedclaimid.md

```diff
@@ -23,27 +23,64 @@ An `XChainCreateClaimID` transaction is used to create a new `XChainOwnedClaimID
 
 ```json
 {
-  "Account": "rHiNUTpDAiQLxMeg2Y9qbQYery1nK1MEmr",
-  "Flags": 0,
-  "LedgerEntryType": "XChainOwnedClaimID",
-  "OtherChainSource": "raromgoXexgo2RnGBRchgL11bZdXdGagaC",
-  "OwnerNode": "0",
-  "PreviousTxnID": "E134CEC9B8799D6AE27011457C69D611ACC4FDD0772AAFFF3B85095EF0195572",
-  "PreviousTxnLgrSeq": 161,
-  "SignatureReward": "304",
-  "XChainBridge": {
-    "IssuingChainDoor": "rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh",
-    "IssuingChainIssue": {
-      "currency": "XRP"
+  "FinalFields": {
```

Comment:
You should remove the `FinalFields` part - that's just from a transaction metadata. These objects are short lived so it's difficult to show what they'd look like, especially with only a few attestations - most of the time have 0 because the attestations come through very quickly.